src/com/projectkorra/projectkorra/command/HelpCommand.java

```diff
@@ -140,21 +141,28 @@ else if (args.size() == 0) {
 			if (!ability.getInstructions().isEmpty()) {
 				sender.sendMessage(ChatColor.GOLD + usage + ability.getInstructions());
 			}
+			if (ability instanceof AddonAbility) {
+				AddonAbility abil = (AddonAbility) CoreAbility.getAbility(arg);
+				
+				sender.sendMessage(color + "* ADDON ABILITY *");
+				sender.sendMessage(ChatColor.GRAY + "- By " + abil.getAuthor());
+			}
+			
 		} else if (Arrays.asList(Commands.airaliases).contains(args.get(0))) {
 			sender.sendMessage(Element.AIR.getColor() + air.replace("/b help AirCombos", Element.AIR.getSubColor() + "/b help AirCombos" + Element.AIR.getColor()));
-			sender.sendMessage(ChatColor.YELLOW + learnMore + ChatColor.DARK_AQUA + "http://tinyurl.com/qffg9m3");
```

Comment:
These changes essentially make the message `Learn more: ` with no link to anything. Either the link needs to be fixed or the message needs to be changed.